root/static/scripts/common/constants.js

```diff
@@ -44,164 +44,165 @@ export const DARTIST_ID = 2;
+  musopen: {host: /^musopen\.org$/},
+  muziekweb: {host: /^www\.muziekweb\.nl$/},
+  muzikum: {host: /^(?:[^/]+\.)?muzikum\.eu$/},
+  myspace: {host: /^myspace\.com$/},
+  napster: {host: /^[a-z]{2}\.napster\.com$/},
+  ndl: {host: /^(?:www\.)?iss\.ndl\.go\.jp$/},
+  niconicovideo: {host: /^(?:ch|www)\.nicovideo\.jp$/},
+  ocremix: {host: /^ocremix\.org$/},
+  offiziellecharts: {host: /^www\.offiziellecharts\.de$/},
+  onlinebijbel: {host: /^www\.online-bijbel\.nl$/},
+  openlibrary: {host: /^openlibrary\.org$/},
+  operabase: {host: /^operabase\.com$/},
+  overture: {host: /^overture\.doremus\.org$/},
+  patreon: {host: /^www\.patreon\.com$/},
+  paypal: {host: /^www\.paypal\.me$/},
+  petitlyrics: {host: /^petitlyrics\.com$/},
+  pinterest: {host: /^www\.pinterest\.com$/},
+  piosenki: {host: /^(?:www\.)?bibliotekapiosenki\.pl$/},
+  progarchives: {host: /^www\.progarchives\.com$/},
+  psydb: {host: /^(?:www\.)?psydb\.net$/},
+  qobuz: {host: /^(?:www\.)?qobuz\.com$/},
+  quebecinfomusique: {host: /^www\.qim\.com$/},
+  rateyourmusic: {host: /^rateyourmusic\.com$/},
+  recochoku: {host: /^recochoku\.jp$/},
+  residentadvisor: {host: /^ra\.co$/},
+  reverbnation: {host: /^www\.reverbnation\.com$/},
+  ric: {host: /^www\.rockinchina\.com$/},
+  rockcomar: {host: /^rock\.com\.ar$/},
+  rockensdanmarkskort: {host: /^www\.rockensdanmarkskort\.dk$/},
+  rockipedia: {host: /^www\.rockipedia\.no$/},
+  rolldabeats: {host: /^(?:www\.)?rolldabeats\.com$/},
+  runeberg: {host: /^runeberg\.org$/},
+  secondhandsongs: {host: /^secondhandsongs\.com$/},
+  setlistfm: {host: /^(?:[^/]+\.)?setlist\.fm$/},
+  smdb: {host: /^(?:www\.)?smdb\.kb\.se$/},
+  snac: {host: /^snaccooperative\.org$/},
+  songfacts: {host: /^(?:[^/]+\.)?songfacts\.com$/},
+  songkick: {host: /^www\.songkick\.com$/},
+  soundcloud: {host: /^soundcloud\.com$/},
+  spiritofmetal: {host: /^(?:www\.)?spirit-of-metal\.com$/},
+  spiritofrock: {host: /^(?:www\.)?spirit-of-rock\.com$/},
+  spotify: {host: /^open\.spotify\.com$/},
+  stage48: {host: /^(?:www\.)?stage48\.net$/},
+  theatricalia: {host: /^(?:www\.)?theatricalia\.com$/},
+  thedancegypsy: {host: /^(?:www\.)?thedancegypsy\.com$/},
+  thesession: {host: /^thesession\.org$/},
+  tipeee: {host: /^www\.tipeee\.com$/},
+  touhoudb: {host: /^touhoudb\.com$/},
+  traxsource: {host: /^www\.traxsource\.com$/},
+  triplejunearthed: {host: /^(?:www\.)?triplejunearthed\.com$/},
+  trove: {host: /^(?:trove\.)?nla\.gov\.au$/},
+  tunearch: {host: /^(?:www\.)?tunearch\.org$/},
+  twitch: {host: /^www\.twitch\.tv$/},
+  twitter: {host: /^twitter\.com$/},
+  utaitedb: {host: /^utaitedb\.net$/},
+  utamap: {host: /^(?:[^/]+\.)?utamap\.com$/},
+  utanet: {host: /^www\.uta-net\.com$/},
+  utaten: {host: /^utaten\.com$/},
+  vgmdb: {host: /^vgmdb\.net$/},
+  viaf: {host: /^viaf\.org$/},
+  videogamin: {host: /^(?:www\.)?videogam\.in$/},
+  vimeo: {host: /^vimeo\.com$/, path: /^\/(?!(?:ondemand|store\/ondemand))/},
```

Comment:
It is probably fine not to match `store/ondemand` here, or it should be matched in `lib/MusicBrainz/Server/Data/URL.pm` too.

```suggestion
  vimeo: {host: /^vimeo\.com$/, path: /^\/(?!(?:ondemand))/},
```

Reply:
This is not matching it, this is avoiding matching it so that it doesn't show the favicon while it gets rejected.